pkg/models/routers/routers.go

```diff
@@ -376,6 +377,17 @@ func (c *routerOperator) createOrUpdateRouterWorkload(namespace string, publishS
 	return nil
 }
 
+func (c *routerOperator) appendArg(args []string, arg string) []string {
+	n := strings.Trim(strings.Split(arg, "=")[0], " \t\r\n")
```

Comment:
It looks like has the same function with `strings.TrimSpace`

Reply:
Thanks for correcting